unbxdAutosuggest.js

```diff
@@ -1281,6 +1329,25 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 			}
 
 		}
+		, sortSuggestionsBylength: function () {
+			this.currentResults.SORTED_SUGGESTIONS = this.currentResults.KEYWORD_SUGGESTION.concat(this.currentResults.TOP_SEARCH_QUERIES);
+			this.currentResults.SORTED_SUGGESTIONS.sort(function (a, b) {
```

Comment:
Hoist common `sort` function and use it. Same code has been used twice.

Reply:
Created common customSort function